### ISSUE TITLE
Change links on EUL webpages

### ISSUE THREAD

**Issue by erussey** [emory-libraries/aspace#23]
Here's the list of webpages that need to be changed:
https://libraries.emory.edu/tools (plus the tools menu)
https://metadata.emory.edu/systems/finding-aids.html
https://oxford.library.emory.edu/archives/collections
https://search.libraries.emory.edu/

The rest seem to be on libguides which is out of scope for this ticket, though I would appreciate it if there is some kind of URL search that could be done to find others.  The EFA link is https://findingaids.library.emory.edu

**Comment by tmill29** [comment]
This ticket cannot begin until June 26th

**Comment by CB987** [comment]
Sorry, @erussey can you clarify what the link should be changed to? Thanks!

**Comment by erussey** [comment]
Yep!  @CB987 : it's https://archives.libraries.emory.edu

**Comment by eporter23** [comment]
@erussey I updated and published changes to the metadata site page. https://metadata.emory.edu/systems/finding-aids.html

**Comment by CB987** [comment]
on libraries: 
the search block Finding Aids destination ✅ PR [#779](https://github.com/emory-libraries/d9-emory-libraries/pull/779)
The following were content changes completed on prod.libraries:
Emory Finding Aids page (node 2331) changed  ✅
https://libraries.emory.edu/rose/using-rose-library/preparing-research-visit#230548828-863258049 ✅
https://libraries.emory.edu/rose/using-rose-library/preparing-research-visit/requesting-materials-rose-library (3 places)✅
https://libraries.emory.edu/rose/collections/about-collection/transferring-records-archives✅
https://libraries.emory.edu/health/using-library/historical-collections ✅
On Oxford.library (changes made on prod.oxford.library) 
https://oxford.library.emory.edu/archives/collections ✅
Main Menu Navigation Archives dropdown ✅

**Comment by CB987** [comment]
changing estimate to a 3 as i didn't realize quite how much hunting there would be ;)

**Comment by CB987** [comment]
On search:
In the accordion dropdowns on the homepage. PR [#1354](https://github.com/emory-libraries/blacklight-catalog/pull/1354)

**Comment by CB987** [comment]
Please note that for the header-embedded search block on libraries/oxford.library, that change is in the code base and thus has to go through the release/QA process before it goes live on stage, and the accordion dropdown link on the home page of search is the same way (basically the ones with PRs), so those are in process but not in place.

**Comment by erussey** [comment]
@CB987 : Did you get the main tools page changed as well?  I wasn't sure if you also changed all the pages in the original ticket + the ones mentioned in the comments.

**Comment by erussey** [comment]
Clarifying the request, which wasn't clear in the original ticket.  We need to change the links and the link text that refers to the system EmoryFindingAids.  Text about finding aids is fine (ArchivesSpace still offers discovery of finding aids), but if text refers to EmoryFindingAids or the EmoryFindingAids database, it needs to be changed to ArchivesSpace.  Happy to clarify text for each page as listed in the original ticket plus the others @CB987 identifed in the comments if that would be helpful.

**Comment by CB987** [comment]
I think I have made all the changes to the link text as well, but any time in the future you happen to see one, just let me know. 
Thanks!

**Comment by erussey** [comment]
@CB987 : Can this be tested?

**Comment by erussey** [comment]
A couple of fixes, which I've broken into separate comments so easier to understand:

- https://libraries.emory.edu/rose/using-rose-library/preparing-research-visit/requesting-materials-rose-library in the first paragraph EmoryFindingAids needs to be changed to "This can be done through either Emory's book catalog, Library Search, or Emory's Finding Aids database, ArchivesSpace" (with ArchivesSpace hyperlinked)

**Comment by erussey** [comment]
https://libraries.emory.edu/tools
![Screenshot 2023-07-05 at 2.02.59 PM.png](https://images.zenhubusercontent.com/62ff941ccf40d827200a9edf/760d95bd-64c2-41a5-bbd4-24577d80d865) Should be changed to Emory Finding Aids (with no spaces) [header] ArchivesSpace provides centralized access to detailed descriptions of archival and manuscript collections held in various repositories at Emory.

**Comment by erussey** [comment]
- https://search.libraries.emory.edu/ ![Screenshot 2023-07-05 at 2.05.41 PM.png](https://images.zenhubusercontent.com/62ff941ccf40d827200a9edf/6a57238f-11a4-448c-9a00-f591692ee4a5) Header should be Emory Finding Aids. Text should be "ArchivesSpace provides centralized access to detailed descriptions of archival and manuscript collections held in various repositories at Emory."  Button should point to https://archives.libraries.emory.edu

**Comment by CB987** [comment]
The search change has been made on GitHub but needs to go through production deploy in order to be visible on the site.
Sorry about the others I missed, I thought I had gotten everything but I will address them ASAP. Thanks for the very specific breakdown!

**Comment by CB987** [comment]
![Screenshot 2023-07-05 at 4.40.13 PM.png](https://images.zenhubusercontent.com/5d379c9d5221e52414fca85c/5a778577-d10f-4f76-9b04-835f4eac136c)
@erussey updated dropdown on search.libraries homepage. you can see the url that the button goes to in the bottom left corner. I will ask for a production deploy whenever you approve. thanks!

**Comment by CB987** [comment]
changing to a 5 as I have found more links and text on the website as well. 
- https://libraries.emory.edu/rose/research-learning/research-services
- https://libraries.emory.edu/rose/research-and-learning/research-services/reprographic-services
- https://libraries.emory.edu/materials/archives-and-special-collections
- https://libraries.emory.edu/rose/research-and-learning/teaching-and-learning-rose-library/students/student-faqs
- https://libraries.emory.edu/rose/using-rose-library/preparing-research-visit/making-or-changing-appointment
- https://libraries.emory.edu/rose/using-reading-room
- https://libraries.emory.edu/rose/using-rose-library/preparing-research-visit/requesting-materials-rose-library
- https://libraries.emory.edu/rose/research-learning/teaching-and-learning-rose-library/students/student-faqs

Can this be tested? - I am not sure if it can be tested exactly but to double-check, the way I looked for things on the website was by using the "this site" search for "EmoryFindingAids" and changed them all-- as of right now I am not getting any results for libraries. i also searched in the backend codebase for "findingaids" and changed them, and am currently getting none. For the websites content that is the only way I can think of to search for the links, but it did find several. One could also look in the github repositories and search for "findingaids" [https://github.com/search?q=org%3Aemory-libraries%20findingaids&type=code] and see if there's any that should have been changed that haven't. (I don't see any from libraries or search.)

requesting-materials-rose-library first paragraph ✅
/tools/ ✅
search change is in the comment above and awaits your approval.

Thanks!

**Comment by erussey** [comment]
@CB987 : Thanks so much...I'll go poke around on the sites you've changed.  Are there any related deploys to these changes (i.e. LibrarySearch)?  If so, are those in place now or do they need to be scheduled?

**Comment by CB987** [comment]
yes, search has to be deployed, and i'll have to defer to tricia on getting that deployed. @erussey everything we put on libraries and oxford.library have been deployed.

**Comment by erussey** [comment]
@CB987 @lovinscari : I've checked all the links above with the exception of Library Search, which has not been deployed.  Once deployed it should look like above and link to https://archives.libraries.emory.edu .  Otherwise this ticket is complete.

**Comment by CB987** [comment]
@erussey search should be deployed also now.

**Comment by erussey** [comment]
And library search looks good.  Signing off on this one @lovinscari .
